lua/plenary/path.lua

```diff
@@ -315,7 +318,9 @@ function Path:is_file()
 end
 
 function Path:is_absolute()
-  -- TODO(windows)
+  if self._sep == '\\' then
+    return string.match(self.filename, '^[A-Z]:\\.*$')
```

Comment:
I think windows has `A:\` -> `Z:\` i am not sure if there are more "labels"

Reply:
Nope as far I know you are right